rest/taskana-rest-spring/src/main/java/pro/taskana/rest/AccessIdController.java

```diff
@@ -66,4 +75,18 @@ public static void setLdapCache(LdapCache cache) {
         ldapCache = cache;
     }
 
+    private List<AccessIdResource> validateAccessId(String accessId) throws InvalidArgumentException {
```

Comment:
does this really validate something? It actually searches all users and groups matching the String, doesn't it? I prefer to have more meaningful names. The code itself is good.

Reply:
This method just tests if the accessId is valid. Return a boolean